custom_components/spotcast/spotify/account.py

```diff
@@ -934,50 +934,30 @@ async def async_view(
             - list: A list of playlists.
         """
 
-        params = {
-            "content_limit": limit,
-            "locale": locale,
-            "platform": "web",
-            "types": "album,playlist,artist,show,station",
-            "limit": limit,
-            "offset": 0,
-        }
+        # Internal method, so that the _get method can be called
+        # As the pager parameters do not allign with the _get parameters
+        # Spotipy _get is defined as: def _get(self, url, args=None, payload=None, **kwargs)
+        def fetch_page(limit, offset):
```

Comment:
I try to avoid nested function they can integrate some nasty bug very hard to test with unit testing.

This could simply be a private method (prepend an underscore to your function) at the same level. I don,t see the need for it to be nested.

Also, I will either need some examples of response in order to write the unit-tests for these methods.